README.md

```diff
@@ -20,38 +20,42 @@ Use this plugin to improve onboarding and HR processes. It adds a Welcome Bot th
 To configure the Welcome Bot, edit your `config.json` file with a message you want to send to a user in the following format:
 
 ```
-"WelcomeMessages": [
-    {
-        "TeamName": "your-team-name",
-        "DelayInSeconds": 3,
-        "Message": [
-            "Your welcome message here. Each list item specifies one line in the message text."
-        ]
-        "AttachmentMessage": [
-            "Attachment message containing user actions"
-        ],
-        "Actions" : [
+"Plugins": {
+    "com.mattermost.welcomebot": {
+        "WelcomeMessages": [
             {
-                "ActionType": "button",
-                "ActionDisplayName": "User Action",
-                "ActionName": "action-name",
-                "ActionSuccessfulMessage": [
-                    "Message posted after the user takes this action and joins channels specified by 'ChannelsAddedTo'."
+                "TeamName": "your-team-name",
+                "DelayInSeconds": 3,
+                "Message": [
+                    "Your welcome message here. Each list item specifies one line in the message text."
+                ],
+                "AttachmentMessage": [
+                    "Attachment message containing user actions"
+                ],
+                "Actions" : [
+                    {
+                        "ActionType": "button",
+                        "ActionDisplayName": "User Action",
+                        "ActionName": "action-name",
+                        "ActionSuccessfulMessage": [
+                            "Message posted after the user takes this action and joins channels specified by 'ChannelsAddedTo'."
+                        ],
+                        "ChannelsAddedTo": ["channel1", "channel2"]
+                    },
+                    {
+                        "ActionType": "automatic",
+                        "ChannelsAddedTo": ["channel3", "channel4"]
+                    }
                 ]
-                "ChannelsAddedTo": ["channel1", "channel2"],
-            },
-            {
-                "ActionType": "automatic",
-                "ChannelsAddedTo": ["channel3", "channel4"]
             }
         ]
     }
-]
+},
 ```
 
 where
 
-- **TeamName**: The team for which the Welcome Bot sends a message for. Refers to the team handle used in the URL. For example, in the following URL the **TeamName** value is `myteam`: https://example.com/myteam/channels/mychannel
+- **TeamName**: The team for which the Welcome Bot sends a message for. Must be the team handle used in the URL, in lowercase. For example, in the following URL the **TeamName** value is `myteam`: https://example.com/myteam/channels/mychannel
```

Comment:
@jasonblais Can we also add the same lowercase info for the channels variables? And replace spaces with hyphens?

Reply:
Good call, updated accordingly